packages/data-service/src/instance-detail-helper.ts

```diff
@@ -415,6 +417,7 @@ export function adaptCollectionInfo({
     view_on: viewOn ?? null,
     pipeline: pipeline ?? null,
     clustered: clusteredIndex ? true : false,
+    fle: encryptedFields ? true : false,
```

Comment:
tiny nit, maybe we could call this `fle2` rather than `fle`, since it won’t apply for FLE 1.0 collections?